pkg/helm/schema_test.go

```diff
@@ -0,0 +1,54 @@
+package helm
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMixin_PrintSchema(t *testing.T) {
+	m := NewTestMixin(t)
+
+	err := m.PrintSchema()
+	require.NoError(t, err)
+
+	gotSchema := m.TestContext.GetOutput()
+
+	wantSchema, err := ioutil.ReadFile("testdata/schema.json")
+	require.NoError(t, err)
+
+	assert.Equal(t, string(wantSchema), gotSchema)
+}
+
+func TestMixin_ValidatePayload(t *testing.T) {
+	testcases := []struct {
+		name  string
+		step  string
+		pass  bool
+		error string
+	}{
+		{"install", "testdata/install-input.yaml", true, ""},
+		{"status", "testdata/status-input.yaml", true, ""},
+		{"upgrade", "testdata/upgrade-input.yaml", true, ""},
+		{"uninstall", "testdata/status-input.yaml", true, ""},
+		{"install.missing-desc", "testdata/bad-install-input.missing-desc.yaml", false, "install.0.helm.description: String length must be greater than or equal to 1"},
```

Comment:
These negative tests are extremely important for json schema validation. Mostly to validate that our schema actually works because schema _passes by default_. I need another pass once we are sure this is all correct to fill in a slew of failure edge case tests for every single property if we really want to know if they are working as we expect.

Reply:
_schema passes by default_

Computers were a mistake.